searx/settings.yml

```diff
@@ -323,6 +323,7 @@ engines:
     timeout: 4.0
     shortcut: apkm
     disabled: true
+    inactive: true
```

Comment:
Should we remove the `disabled: true` when we set `inactive: true`?

Reply:
It's a good idea yes.

Reply:
I did comment the engine where "disabled: true" is found. This way we keep the history for engines that were disabled.